runtime/tutorials/en/01-vim-beginner.tutor

```diff
@@ -0,0 +1,959 @@
+~~~
+
+# Lesson 7.1: GETTING HELP
+
+** Use the on-line help system **
+
+Vim has a comprehensive on-line help system.  To get started, try one of
+these three:
+    - press the `<HELP>`{normal} key (if you have one)
+    - press the `<F1>`{normal} key (if you have one)
+    - type
+        `:help`{vim}
+
+Read the text in the help window to find out how the help works.
+Type `<Ctrl-w><Ctrl-w>`{normal} to jump from one window to another.
+Type `:q`{vim} to close the help window.
+
+You can find help on just about any subject, by giving an argument to the
+":help" command.  Try these (don't forget pressing <Enter>):
+~~~ cmd
+    :help w
+    :help c_CTRL-D
+    :help insert-index
+    :help user-manual
+~~~
+# Lesson 7.2: CREATE A STARTUP SCRIPT
+
+** Enable Neovim features **
+
+Neovim has many more features than Vi (and Vim), but some of them are disabled by
+default.  To start using more features you have to create a "nvimrc" file.
+
+  1. Start editing the "vimrc" file.  This depends on your system:
+    `:e ~/.nvimrc`{vim}     for Unix-like systems
+    `:e $VIM/_nvimrc`{vim}      for Microsoft Windows
+
+  2. Now read the example "vimrc" file contents:
+    `:r $VIMRUNTIME/vimrc_example.vim`{vim}
+
+  3. Write the file with:
+    `:w`{vim}
+
+  The next time you start Neovim it will use syntax highlighting.
+  You can add all your preferred settings to this "nvimrc" file.
+  For more information type  :help vimrc-intro
+
+# Lesson 7.3: COMPLETION
+
+** Command line completion with `<Ctrl-d>`{normal} and `<Tab>`{normal} **
+
+  1. Look what files exist in the directory:  `:!ls`{vim}
+
+  2. Type the start of a command:  `:e`{vim}
+
+  3. Press `<Ctrl-d>`{normal} and Neovim will show a list of commands that start with "e".
+
+  4. Press `<Tab>`{normal} and Neovim will complete the command name to ":edit".
+
+  5. Now add a space and the start of an existing file name: `:edit FIL`{vim}
+
+  6. Press `<Tab>`{normal}.  Neovim will complete the name (if it is unique).
+
+NOTE:  Completion works for many commands. It is especially useful for `:help`{vim}.
+
+# Lesson 7 SUMMARY
+
+  1. Type `:help`{vim} 
+     or press `<F1>`{normal} or `<Help>`{normal} to open a help window.
+
+  2. Type  `:help TOPIC`{vim} to find help on TOPIC.
+
+  3. Type  `<Ctrl-w><Ctrl-w>`{normal}  to jump to another window
+
+  4. Type  `:q`{vim}  to close the help window
+
+  5. Create a nvimrc startup script to keep your preferred settings.
+
+  6. While in command mode, press `<Ctrl-d>`{normal} to see possible completions.
+     Press `<Tab>`{normal} to use one completion.
+
+# CONCLUSION
+
+This was intended to give a brief overview of the Vim editor, just enough to
+allow you to use the editor fairly easily. It is far from complete as Vim has
+many many more commands.  Consult the help often.
+
+Now, you can try a different tutorial:
+
+    `:Tutor 02-vim-intermediate`{vim}
+
+There are many resources online to learn more about vim. Here's a list of a bunch of them:
```

Comment:
suggestion: `Here's a list of a bunch of them` => `Here's a bunch of them` or `Here's a list of them`